pyProbeParticle/ocl/AFMulator.py

```diff
@@ -36,7 +37,7 @@ class AFMulator():
         iZPPs: int. Element of probe particle.
         QZs: Array of length 4. Position tip charges along z-axis relative to probe-particle center in angstroms.
         Qs: Array of length 4. Values of tip charges in units of e. Some can be zero.
-        rho: Dict or MultipoleTipDensity. Tip charge density. Used with Hartree potentials. Overrides QZs and Qs.
+        rho: Dict or :class:`.MultipoleTipDensity`. Tip charge density. Used with Hartree potentials. Overrides QZs and Qs.
```

Comment:
I have seen, that everything is not affecting the running part of the code. These are only comment, I understand that adding the `:class:` is required by the automatic documentation, on the other hand it is pretty confusing for reader, I think. Is there some other way around?

Reply:
These directives (`:class:`, etc) are not strictly necessary for the documentation, but they are just a convenience thing that adds a clickable link into the html page to take you to the definition of the class. The idea in the end is that you would not need to go to the source code to read these comments, but you would rather just go to the webpage where it looks nice once we get it up and running. Of course it's a bit more work and it's probably something we should discuss in a meeting if we want to have it. I would anyway merge it like this for now as an example of things that you can do.

Reply:
Ok, I see, just asking. Good to know.